### ISSUE TITLE
Provide context local for file reference

### ISSUE THREAD

**Issue by SevereOverfl0w** [carrot/roots-browserify#23]
Would it be within the scope of this extension to do something similar to what `js_pipeline` does and provide a `js` function, which will produce the code required to display the script in the browser?

**Comment by jenius** [comment]
Yes, absolutely. I'll merge this and ship right away if you PR this in. It should be something like `browserify()` though, so as not to duplicate the `js` function in case for some reason someone is using both extensions :grinning:
